### PR TITLE
node: add gcloud to 8-user and 10-user

### LINKED ISSUE #13: Nodejs containers lack cloud-sdk.
## Problem

Nodejs containers do not have gcloud installed.

## Solution

Add gcloud with beta component to the Dockerfile for Node 8 and Node 10.
This would match observed state in Java, Go, and Python.

### PR DESCRIPTION
**Pull request by grayside**
Fixes #13

### CODE REVIEW

**Comment by grayside** [comment]
After discussion with @fhinkel, we found the nodejs-docs-samples did not have gcloud available, so it seems any sample tests relying on gcloud are silently failing.

**Comment by busunkim96** [comment]
LGTM.

@bcoe Could you take a look at the changes here?

**Comment by fhinkel** [comment]
Thanks!

File: node/10-user/Dockerfile [hunk 1]
```diff
@@ -28,3 +28,19 @@ ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
 
 RUN pyenv install 3.7.1
 RUN pyenv global 3.7.1
+
+# Setup Cloud SDK
+WORKDIR /home/node
```
Comment (bcoe) [inline node/10-user/Dockerfile]: will `WORKDIR` have an effect on the default entry point for the container? I think currently it's a assumed to be `/usr/local/bin`:

https://github.com/nodejs/docker-node/blob/f8f2384f7edc345f5ffc0496458005981b512882/10/stretch/Dockerfile#L64

Is there an easy way to ensure that we leave `WORKDIR` the default value.

Comment (grayside) [reply node/10-user/Dockerfile]: WORKDIR does indeed set the default entry working directory, but that COPY statement does not. The WORKING directory before this change was `/`, I can add another WORKDIR at the end to set it back to that value.

**Comment by busunkim96** [comment]
[Build ](https://fusion.corp.google.com/invocation/a2c945cd-3f33-4bf0-823f-a590953c2903?search_pattern=testing-infra-docker&include_inactive_projects=false&target=cloud-devrel%2Fclient-libraries%2Ftesting-infra-docker%2Frelease%2Fnode&drilldownTab=tests) to update the image. It took ~20 minutes last time, but it may take longer.

You'll see the new images here: https://pantheon.corp.google.com/gcr/images/cloud-devrel-kokoro-resources/GLOBAL/node?project=cloud-devrel-kokoro-resources&folder&organizationId&gcrImageListsize=30

**Comment by grayside** [comment]
It looks like the build completed, but 1) only built node:8-pupeteer, and 2) did not push the resulting image to gcr, meaning it did not use .kokoro/build.sh.

**Comment by busunkim96** [comment]
I believe they've been pushed now. 

The logs are a little hard to read on fusion. Here are the [cloud build logs](https://pantheon.corp.google.com/cloud-build/builds/4e257510-abb7-43ab-b28b-47b4e5e06ad8?project=cloud-devrel-kokoro-resources), which has one log per image.

**Pull request merged**